Firestore/core/src/firebase/firestore/core/field_filter.h

```diff
@@ -74,11 +72,24 @@ class RelationFilter : public Filter {
   bool IsInequality() const override;
 
  protected:
-  bool Equals(const Filter& other) const override;
+  /**
+   * Creates a new filter that compares fields and values. Only intended to be
+   * called from Filter::Create().
+   *
+   * @param field A path to a field in the document to filter on. The LHS of the
+   *     expression.
+   * @param op The binary operator to apply.
+   * @param value_rhs A constant value to compare @a field to. The RHS of the
```

Comment:
Nit: do we use `@a` tags in iOS doc comments?

Reply:
It's more common in google3, but I think it's obsolescent here. Changed to backticks.

There are several other instances in the codebase that I'm not addressing as a part of this PR.